test/common/helpers/login-helpers.unit.spec.js

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { handleVerify } from '../../../src/js/common/helpers/login-helpers.js';
+
+// Does this need to be a React class in order for something to magically call this.serverRequest()?
```

Comment:
I don't think `this.serverRequest` is a function, it looks like it's a fetch promise that sometimes has `abort` called on it in RequiredLoginView. Though I can't tell if that would every apply to the verify call. I also didn't know `abort` was something you could call on a promise?